dotcom-rendering/src/web/components/Caption.tsx

```diff
@@ -239,6 +247,9 @@ export const Caption = ({
 	if (noCaption && (noCredit || hideCredit)) return null;
 
 	const palette = decidePalette(format);
+	const isLiveblog =
```

Comment:
We’ve picked up `isBlog` recently, might be a valid option here.

```suggestion
	const isBlog =
```

Reply:
Thats actually a much better name, thanks!